registry-requirements.md

```diff
@@ -13,112 +12,130 @@ From the charter:
+  - Ege: We are checking basic implementability, not conformance or certification. This requires implementation experience (test suite, virtual or physical plugfest, etc.)
+  - Koster: We won't be able to do a "lab test" for certification (e.g., Matter). The assertions to satisfy will be generic to all bindings.
+  - Cris: Testing the binding without a F2F event should be possible.
+  - Jan: Should there be interop testing? How many Thing implementations from the submitter? How many Consumer implementations from the submitter? What kind of implementation from non-submitter?
+  - Implementation Experience:
+    - Ege, Koster: The submitters do the following: In an event (not necessarily a W3C event), each operation of the binding is executed automatically (testing the protocol driver and TD parsing), and an example logic execution (not necessarily code) is provided together with information of the testing environment (devices, Consumer application). This event CAN be VPN-based. It MUST contain at least two separate entities where one Thing and one Consumer in total are present, e.g., entity A providing the Thing and entity B providing the Consumer application.
+      - Ideally, the custodian, reviewers and submitters would collaborate in a Plugfest on the implementation and collect experience together.
+      - Cris: This testing event can continue to collect more inputs on the binding and we showcase these results in a dashboard/report dynamically.
+      - Daniel: The kind of testing that happened should be documented and can happen over time.
+      - DISCUSS: Should we enforce two separate entities?
+        - We can require one Thing and Consumer but collect the number of independent implementations and show it in the registry table. More implementations would show more maturity of the binding.
+10. The binding MUST contain the following sections in the order presented below. The binding CAN contain other sections anywhere, including between the required ones. The submitters are encouraged to look at the existing submissions. There MUST be at least one operation mapped to a protocol message/vocabulary term. The submitter SHOULD use the table template provided in the document for the vocabulary.
+  - Introduction
+  - Binding Content:
+    - URL Format
+    - Form Vocabulary Definition as Table
+    - Default and possible mappings to operations as a Table
+  - Examples
+Opinions:
+  - Cris: Fine. Maybe more sections make sense. To be decided later.
+  - Kaz: mention the example bindings as a way how the submitter can write a binding.
+11. DISCUSS: What are the required (machine-readable) documents provided on the side?
+  - Ege: JSON Schema, Ontology File (RDF and HTML) not mandatory?, test report or documentation of implementation experience
+12. DISCUSS: Where should discussions on implementation experience be collected?
+13. DISCUSS: How to handle overlaps between two bindings in the review process?
```

Comment:
I was wondering whether the organization that publishes/developed the protocol in question could endorse a certain protocol binding, and that endorsement could then be marked in the table as well (with a checkmark, for example). E.g., OASIS could endorse an/our MQTT binding, making it the “official” one, but there could also be alternative binding documents for MQTT without an endorsement listed in the table.

Reply:
I would avoid having two MQTT bindings, one endorsed one not.

Reply:
Hmm, yeah, you are right of course, I just had the [comment](https://github.com/w3c/wot-binding-templates/issues/393#issuecomment-2622612507) by @benfrancis in https://github.com/w3c/wot-binding-templates/issues/393 in mind here. So essentially, I was wondering whether the authoring organization should also have a say here, or whether the group could also decide to ignore the SDO and recommend a different binding that satisfies the requirements better, for example.

Reply:
If the custodian chooses a reviewer that decides on a _better_ one, we have to go with that better one by our rules. So far, we have not declared precedence of an SDO-endorsed over another one. However, given this possibility and comment from @benfrancis, we do have to define what happens in such cases. Created https://github.com/w3c/wot-binding-templates/issues/398 for the discussion. @JKRhb if you are fine with it, let's resolve this comment and continue the discussion there. I am doing that for all discussion points in this pr.